setup.py

```diff
@@ -14,7 +14,7 @@
     license='MIT',
     description='SendGrid Backend for Django',
     long_description=open('./README.rst').read(),
-    install_requires=["sendgrid >= 3.5, < 4"],
+    install_requires=["sendgrid >= 3.5, < 4.0.4"],
```

Comment:
Yes, I understand that the python-sendgrid package has hit version 4. I also understand that we _previously did not allow_ django-sendgrid to be installed alongside python-sendgrid version 4.

This change now allows for django-sendgrid to be installed along python-sendgrid version 4. Does django-sendgrid actually support python-sendgrid version 4?